nuget/Microsoft.Windows.CppWinRT.props

```diff
@@ -6,6 +6,12 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 <!--Set compiler and linker options for projects requiring C++/WinRT. -->
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
+    <PropertyGroup>
+        <!-- Only do this for MSBuild versions below 16.0
+             as it is since done automatically, see https://github.com/microsoft/msbuild/pull/3605-->
+        <MSBuildAllProjects Condition="'$(MSBuildToolsVersion)'  &lt;= '15'">$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
```

Comment:
This should be ok. We added `$(MSBuildVersion)` in 15.6, which is a little clearer, but it wasn't supported in 15.0 which is still in support for a little while.